### PR TITLE
Presto to trino include plugin

### PR DESCRIPTION
**Pull request by gugalnikov**
## Closes/fixes/resolves issue(s)?
#106 
## What was added/changed/fixed?
- all references to Presto have been updated
- image was also fixed to include Trino icon
## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [ ] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### CODE REVIEW

**Review by zhenik — COMMENTED** [review]
So far its good. I will wait for its ready

File: dev/ansible/05_trino_create_tables.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@
     - test
 
 - name: Create CSV table
-  shell: docker run --network host -v "/vagrant/example/resources/query/csv_create_table.sql:/csv_create_table.sql" prestosql/presto:341 presto --http-proxy 127.0.0.1:8888 --server 127.0.0.1:8888 --catalog hive --schema default --user presto --file /csv_create_table.sql
+  shell: docker run --network host -v "/vagrant/example/resources/query/csv_create_table.sql:/csv_create_table.sql" trinodb/trino:354 trino --http-proxy 127.0.0.1:8888 --server 127.0.0.1:8888 --catalog hive --schema default --user trino --file /csv_create_table.sql
```
Comment (claesgill) [inline dev/ansible/05_trino_create_tables.yml:79]: We should consider extracting `trinodb/trino:354` to a ansible variable. Used alot of places.

**Review by claesgill — APPROVED** [review]
Great work. LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,12 @@
 ### Changed
 
 - Prestosql changed name to Trino. This repo changed with it.
+- Updated all references to Presto
```
Comment (claesgill) [inline CHANGELOG.md:8]: Didn't see before now., but can you add issue number?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,10 +5,12 @@
...
 
 ### Added
 
 - Trino consul connect plugin source and build
+- Using Trino docker image with tag 354
```
Comment (claesgill) [inline CHANGELOG.md:13]: Same here.

**Comment by gugalnikov** [comment]
done with the changes proposed by reviewers

**Pull request merged**